core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala

```diff
@@ -390,25 +389,24 @@ trait GitHubResolver {
  * Supports most of the references documented in:
  * https://help.github.com/articles/autolinked-references-and-urls/
  */
-case class GitHubDirective(page: Page, variables: Map[String, String])
+case class GitHubDirective(ctx: Writer.Context)
   extends ExternalLinkDirective("github", "github:") with GitHubResolver {
 
-  def resolveLink(node: DirectiveNode, link: String): Url = {
+  def resolveLink(node: DirectiveNode, link: String): Url = try {
```

Comment:
Unnecessary `try` here, or was it going to catch?

Reply:
An earlier version did catch, but the error collecting moved into the methods being delegated to.